src/audio/drc/ref_sin_fixpt.c

```diff
@@ -35,8 +35,7 @@
  */
 void ref_sine_fixpt(const int32_t thRadFxp[1024], int32_t cdcSinTh[1024])
```

Comment:
commit message should read something like this: `drc: Add lookup table`

See this: https://chris.beams.io/posts/git-commit/#seven-rules

Reply:
for your draft you should replace the sine function in the actual firmware